cli/src/lib.rs

```diff
@@ -426,13 +437,17 @@ pub enum IdlCommand {
         /// Useful for multisig execution when the local wallet keypair is not available.
         #[clap(long)]
         print_only: bool,
+        #[clap(short, long)]
+        priority_fee: Option<u64>,
     },
     /// Command to remove the ability to modify the IDL account. This should
     /// likely be used in conjection with eliminating an "upgrade authority" on
     /// the program.
     EraseAuthority {
         #[clap(short, long)]
         program_id: Pubkey,
+        #[clap(short, long)]
+        priority_fee: Option<u64>,
```

Comment:
Using `short` for both here would panic `clap`. It would be better to remove the `short` argument from the `priority_fee` fields, since it's also ambigious with `--program-name` (`-p`) that's common for other commands.